database/factories/LessonFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Lesson;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class LessonFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Lesson::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'title' => $this->faker->text(),
+            'description' => $this->faker->text(),
+            'requirement' => $this->faker->realText(),
+            'content' => $this->faker->realText(),
+            'course_id' => rand(1, 30),
```

Comment:
course_id không nên fake, vì có fake cũng không chính xác. Cần tạo factory theo relation tốt hơn https://laravel.com/docs/8.x/database-testing#belongs-to-relationships